### ISSUE TITLE
can't select version compiled with specific version of OTP

### ISSUE THREAD

**Issue by simonmcconnell** [version-fox/vfox-elixir#6]
I'm used to using `scoop install elixir@1.16.3-otp-25`.  I don't see the option to specify the OTP version with vfox.

**Comment by yeshan333** [comment]
> I'm used to using `scoop install elixir@1.16.3-otp-25`. I don't see the option to specify the OTP version with vfox.

Yes, this method is not supported at present. If you are interested, you can summit a pull request.

**Comment by simonmcconnell** [comment]
How about a PR switching to use hex.pm builds which will kill a few birds with the one stone?
